config/config.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Hmm, I just matched the copyright in the other files -- not to say those were accurate.

Fixed?